npm-shrinkwrap.json

```diff
@@ -389,7 +389,7 @@
           "dependencies": {
             "nan": {
               "version": "1.3.0",
-              "from": "nan@1.3.0",
+              "from": "https://registry.npmjs.org/nan/-/nan-1.3.0.tgz",
```

Comment:
Probably it would be a good idea to have it, but not in this PR. I'll open an issue for that.